src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs

```diff
@@ -157,12 +179,52 @@ private static ReportType GetReportType(bool isDeprecated, bool isOutdated, bool
             throw new ArgumentException(string.Format(Strings.ListPkg_InvalidOptions));
         }
 
+        private static (IReportRenderer, ReportOutputFormat) GetOutputType(string outputFormatOption, string outputVersionOption)
+        {
+            ReportOutputFormat outputFormat = ReportOutputFormat.Console;
+            if (!string.IsNullOrEmpty(outputFormatOption))
+            {
+                try
+                {
+                    outputFormat = EnumExtensions.GetValueFromName<ReportOutputFormat>(outputFormatOption);
+                }
+                catch (ArgumentException)
+                {
+                    string currentlySupportedFormat = "console, json";
+                    throw new ArgumentException(string.Format(Strings.ListPkg_InvalidOutputFormat, outputFormatOption, currentlySupportedFormat));
+                }
+            }
+
+            if (outputFormat == ReportOutputFormat.Console)
+            {
+                return (ConsoleWriter.Instance, ReportOutputFormat.Console);
+            }
+
+            // currently only version 1 is available, so default to latest available version 1.
+            IReportRenderer jsonReportRenderer;
+
+            var currentlySupportedReportVersions = new List<string> { "1" };
+            // If customer pass unsupported version then default to latest available version and warn about unsupported version.
```

Comment:
The comment here is warn, but we seem to log jsonReportRenderer.WriteErrorLine. 

Shouldn't this be a 1 exit code, invalid input? 
Seems better than ignoring a bad parameter.